models/src/it/java/io/sphere/sdk/utils/CleanupFunction.java

```diff
@@ -0,0 +1,6 @@
+package io.sphere.sdk.utils;
+
+@FunctionalInterface
+public interface CleanupFunction<T> {
```

Comment:
You don't need to declare a functional interface, you could use http://docs.oracle.com/javase/8/docs/api/java/util/function/Consumer.html .